examples/graphql-client-handle-partial-response/graphql_client_handle_partial_response.metatags

```diff
@@ -1,2 +1,2 @@
-description: A GraphQL client written in Ballerina
+description: This example demonstrates handling partial data returned along with errors.
```

Comment:
Shouldn't we mention `GraphQL client` here?

Reply:
Updated.